service/controller/release.go

```diff
@@ -51,7 +51,7 @@ func NewRelease(config ReleaseConfig) (*Release, error) {
 	{
 		c := informer.Config{
 			Logger:  config.Logger,
-			Watcher: config.G8sClient.Release().ReleaseCycles(""),
+			Watcher: config.G8sClient.ReleaseV1alpha1().ReleaseCycles(""),
```

Comment:
This should already make sure that the CRD exists or am I not seeing something? 
`Operatorkit` makes sure that the crd exists for the CR we are watching.

Reply:
This is the ReleaseCycle CRD, but this PR is about Release CRD.